src/rfq/protocols/hashflow/client.rs

```diff
@@ -433,35 +431,68 @@ impl RFQClient for HashflowClient {
 
                     let mut quote_attributes: HashMap<String, Bytes> = HashMap::new();
                     quote_attributes.insert("pool".to_string(), quote.quote_data.pool);
+                    if let Some(external_account) = quote.quote_data.external_account {
+                        quote_attributes.insert("external_account".to_string(), external_account);
```

Comment:
what exactly is the external_account in the end?

Reply:
aaahhhh so we are missing that bit no? to check if it is passed, and if not use the zero address